server/jwt.go

```diff
@@ -82,9 +85,22 @@ func validateToken(jwtKeyFunc keyfunc.Keyfunc, token string, expectedTenantIDs [
 		// There's no WithNotBefore() helper, but the library always verifies if the claim is present.
 	}
 
+	mmServerURL, err := url.Parse(siteURL)
```

Comment:
I don't think we can add a param to the actual site URL of the MM instance.  I was thinking this code path would be called in response to a HTTP request to fetch the iFrame and the request would have the param.  The request param would be passed here via context or a flag added to this method that determines if community audience should be used.

Reply:
I'll update the PR.